lightning/src/util/config.rs

```diff
@@ -224,6 +236,23 @@ pub struct UserConfig {
 	pub peer_channel_config_limits: ChannelHandshakeLimits,
 	/// Channel config which affects behavior during channel lifetime.
 	pub channel_options: ChannelConfig,
+	/// If a channel is not set to be publicly announced, we reject HTLCs which were set to be
```

Comment:
nit: maybe we should dissociate HTLC "rejection" at `update_add_htlc` processing from "forward failure", what do you think about "fail forward" ?

Reply:
You mean in the docs? I feel like "reject" communicates whats going on, especially to a user who doesn't know about internal distinctions of HTLC rejections. "we fail HTLCs which were set" is slightly less clear to me, but also has roughly the same meaning in non-technical english.

Reply:
Well, i think it's yet another anticipation of introducing custom forward policy (`PendingHTLCsRelayable`), which would a new source of rejection cases. We'll cleanup at that time i guess.